src/sentry/api/serializers/models/alert_rule_trigger_action.py

```diff
@@ -23,6 +23,15 @@ def human_desc(self, action):
         elif action.type == action.Type.MSTEAMS.value:
             return "Send a Microsoft Teams notification to " + action.target_display
 
+    def format_target_identifier(self, action):
```

Comment:
@ceorourke Much better! But maybe we should call this function like `get_identifier_from_action` since the action is the input.